DOCS_GUIDELINE.md

```diff
@@ -0,0 +1,72 @@
+# Documentation guideline
+
+## General guidelines
+
+* Always use `hexadecimal` when defining offsets. They can be defined as follows. 
+  * With `0x` before the value: `0x1234`
+  * With `h` after the value: `1234h`
+
+* Although `hexadecimal` is preferred, `decimal` can be used for the rest of values.
+
+---
+
+## Representing single variables
+
+Here's a list of the recommended basic variable types. Size is in *bits*.
+
+| Size | Signed | Unsigned
+|------|--------|----------
+| 8  |  int8  |  uint8  
```

Comment:
I would prefer 'byte', 'short' and 'int' over 'int8', 'int16' and 'int32'

Reply:
The reason why I chose this is because it clearly states the amount of bytes it uses and the type of number it stores: `int8` is an integer 8 bits. `byte` can be confusing as to whether it includes signed values or not where `int8` has 0 ambiguity.

Reply:
I mean, byte is one byte. short is 2 bytes. int is 4 bytes. This must be common knowledge.
Also the ambiguity talk is invalid because if there is no 'u' in front, it is signed.
But we should also get input from the others about this.

Reply:
Depending on compiler and target platform the size of an int can change. Personally I prefer int8, int16 etc over just short, int, long as there's no confusion possible.

Reply:
+1 to Delta's notion.

Reply:
That settles that, I guess.

Reply:
While I think byte,short,int... is easier to read, I agree with Delta, IntX/UintX specifies the exact size of the field, which I think works best for documentation. As a side note, @Truthkey, the same way you have int/uint, you also have sbyte/byte.